### ISSUE TITLE
Possible typo in docstring 

### ISSUE THREAD

**Issue by dvav** [tensorflow/probability#362]
I believe the documentation at this point should use a logit function, not just log.

https://github.com/tensorflow/probability/blob/f2d7ea844ffdf695742661fc8b304320da0bb757/tensorflow_probability/python/distributions/categorical.py#L116

**Comment by brianwa84** [comment]
I think the doc is correct because the logits get transformed to probs via p_i = exp(logit_i) / sum_j(exp(logit_j)), ie softmax.
